tox.ini

```diff
@@ -0,0 +1,8 @@
+[tox]
+envlist = py35, py36, py37
+recreate=true
+skip_missing_interpreters=true
+
+[testenv]
+deps = pytest
```

Comment:
Rather than running pytest, we would prefer running 
black, mypy, flake8, and isort!
And please add these changes to setup.cfg under tox!